src/tcp.c

```diff
@@ -185,9 +201,9 @@ enftun_tcp_connect_any(struct enftun_tcp* tcp,
 }
 
 void
-enftun_tcp_close(struct enftun_tcp* tcp)
+enftun_tcp_native_close(struct enftun_tcp_native* ctx)
```

Comment:
Is close different for `_native` than it will be for `_psock`?  If not, keep it as `enftun_tcp_close(struct enftun_tcp* tcp)`.